### DISCUSSION TITLE
Use OpenWeather Map data as label within pictures?

### DISCUSSION THREAD

**Discussion by danteffm** [aaronwmorris/indi-allsky#1695] [General]
Dear Aaron, first of all: Thank you so much for your amazing software. I'm really a noob when it comes to programming, so please let me tell you that I found the installation and setup very easy to be done. 

Could you please tell me how to include OpenWeather Map data as an image label - if this can be done?  I already to managed to change the labels like described on this page https://github.com/aaronwmorris/indi-allsky/wiki/Image-Labels#system-temperature-sensors but didn't find any hints about OpenWeather Map data.

**Comment by aaronwmorris** [comment]
Thanks!  When you enable the OpenWeather Map API under Sensors, the data will be populated in the User Sensor Slots.  Assuming you assign OpenWeather map data starting at User Slot 10 to slot 18 (9 slots total).  If you wanted to show the humidity on the image, that would be the 3rd (from 10) slot which would be `{sensor_user_12:0.1f}` in the image label.

https://github.com/aaronwmorris/indi-allsky/wiki/Image-Labels#user-sensors

https://github.com/aaronwmorris/indi-allsky/wiki/Sensors#openweather-map

**Comment by danteffm** [comment]
Hey Aaron, amazing - works perfectly! Thank you so much! :-) 

PS: Yeah, i still have to focus the camera a bit better ;-) 
![Bildschirmfoto 2024-12-04 um 17 15 38](https://github.com/user-attachments/assets/90ca190a-9485-4651-b14c-67d64dd94b75)

**Reply by aaronwmorris** [reply]
Very nice!
